src/client/context.rs

```diff
@@ -3,6 +3,8 @@ use std::collections::HashMap;
 use std::fmt::Write as FmtWrite;
 use std::io::Read;
 use std::sync::{Arc, Mutex};
+#[cfg(feature="extras")]
+use std::ops::ShlAssign;
```

Comment:
Place this before the `use super::CACHE` below for consistency (grouping conditional uses together)